smart-contracts/contracts/mixins/MixinERC721Enumerable.sol

```diff
@@ -54,8 +54,7 @@ contract MixinERC721Enumerable is
   ) public view
     returns (uint256)
   {
-    require(_index < balanceOf(_keyOwner) && _keyOwner != address(0), 'ONLY_ONE_KEY_PER_OWNER');
-    return getTokenIdFor(_keyOwner);
+    return getKeyOfOwnerByIndex(_keyOwner, _index);
```

Comment:
Maybe we can remove `tokenOfOwnerByIndex` ?

Reply:
yes, just `tokenOfOwnerByIndex` is part of standard erc721 from OZ